### PR TITLE
feat(fa): Multiple muni & update staff add on muni

### PR DESCRIPTION
**Pull request by MargretFinnboga**
# ... ☝🏼 

https://app.asana.com/0/1202037685216853/1201234405372285

## What

- Endpoint that retrieves get all muni 
- Removes shared hook
- Add use muni hook to veita(gets all muni) and osk(gets one muni) 
- When staff is updated send municipaliy Ids with to update that 

## Why

- Veita and osk are using muni differently now so shared muni hook is outdated 
- We want to be able to add muni to staff 

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/financial-aid/web-veita/src/components/MultiSelection/MultiSelection.css.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { style } from '@vanilla-extract/css'
+import { theme } from '@island.is/island-ui/theme'
+
+export const selectContainer = style({
```
Comment (davidpxn) [inline apps/financial-aid/web-veita/src/components/MultiSelection/MultiSelection.css.ts:4]: Is this necessary?  If we are using these styles for a Box component then we can just send it in as a prop:
display="block"

File: apps/financial-aid/web-veita/src/utils/useMunicipalities.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        livesWithParents
+        unknown
+        type
+      }
+    }
+  }
+`
+
+export const useMunicipalities = () => {
+  const storageKey = 'currentMunicipalities'
```
Comment (davidpxn) [inline apps/financial-aid/web-veita/src/utils/useMunicipalities.ts:42]: SHould this be in constants?

Comment (MargretFinnboga) [reply apps/financial-aid/web-veita/src/utils/useMunicipalities.ts:42]: Its not really used anywhere else, so I am not sure if that is necessary? 🙃

**Review by davidpxn — APPROVED** [review]
Very næs 💁‍♂️

**Pull request merged**